README.rst

```diff
@@ -112,7 +114,7 @@ a single module, you can specify that one specifically
 (ex. ``localStorage.debug = 'flyte:adminEntity'`` to only see decoded Flyte
 Admin API requests).
 
-.. _cors-proxy:
+.. _cors-proxying:
```

Comment:
I guess either GitHub doesn't read this or there is some sort of weird logic. This change basically makes the custom anchor being the same as what is generated by GitHub from section header.

Reply:
Oh huh. That is strange.